source/docs/articles/organizations/change-management.md

```diff
@@ -54,74 +55,127 @@ Change Management is enabled for all Pantheon for Agencies Partners, and for som
+        <td class="border-bottom">- Manage settings<br>
+        - Manage team (add/remove)<br>
+        - Deploy code to Test/Live<br>
+        - Assign a "User in Charge": This is the person who created a site and is able to delete it.</td>
+    </tr>
+    <tr>
+        <td class="border-bottom">Developer</td>
+        <td class="border-bottom">- Deploy code to CDEs and Dev only</td>
+    </tr>
+</table>
 
-<style type="text/css">
-.tg  {border-collapse:collapse;border-spacing:0;}
-.tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
-.tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
-.tg .tg-e3zv{font-weight:bold}
+###Partner Organization - Roles/Permissions
+<style>
+    .data-table {
+        border-collapse: collapse;
+    }
+    .border-bottom {
+        border-bottom: 1px solid #000;
+    }
 </style>
+
 <table class="tg">
-<tr>
-<th class="tg-e3zv">Role</th>
-<th class="tg-e3zv">Organization Dashboard</th>
-<th class="tg-e3zv">Site Dashboard</th>
-</tr>
-<tr>
-<td class="tg-031e">Owner</td>
-<td class="tg-031e">Functionally the same as an Organization Admin, distinguished only by the user/organization's association with a payment instrument.  </td>
-<td class="tg-031e">Same as Organization Administrator.</td>
-</tr>
-<tr>
-<td class="tg-031e">Organization Administrator</td>
-<td class="tg-031e">Only role that can access this dashboard. All permissions are granted.</td>
-<td class="tg-031e">In Pantheon for Enterprise Organizations, the Site is owned by the Organization, and the Organization Administrator functions as site Owner and can:<br>- Manage Sites Service Level<br>- Delete Sites<br>- Manage Settings<br>If the Site is not owned by the Organization, the Organization Administrator functions as a Team Member.</td>
-</tr>
-<tr>
-<td class="tg-031e">Team Member</td>
-<td class="tg-031e">Role does not exist in organizations without Change Management</td>
-<td class="tg-031e">- Manage Settings<br>- Manage Team<br>- Manage Settings<br>- Deploy to Test/Live</td>
-</tr>
-<tr>
-<td class="tg-031e">Unprivileged*</td>
-<td class="tg-031e"> No Access</td>
-<td class="tg-031e">- Become Team Members for sites they create<br>- Can only access sites through invitation, are considered team members.</td>
-</tr>
+    <tr>
+        <th><strong>Role</strong></th>
+        <th><strong>Permissions</strong></th>
+    </tr>
+    <tr>
+        <td>Administrator</td>
+        <td>- Manage settings<br>
+        - Remove members    <br>
+        - Edit member roles <br>
+        - Delete sites  <br>
+        - Remove sites from the Organization</td>
+    </tr>
+    <tr>
+        <td class="border-bottom">Team Member</td>
+        <td class="border-bottom">- Deploy code<br/>- Assign a "User in Charge": This is the person who created a site and is able to delete it.</td>
```

Comment:
This is only called "User in Charge" in enterprise One orgs, I believe? @bensheldon @slivermon may be able to clarify.